src/DynamoRevit/DynamoRevit.cs

```diff
@@ -251,10 +218,6 @@ private static void InitializeCore(ExternalCommandData commandData)
         {
             if (initializedCore) return;
 
-#if !ENABLE_DYNAMO_SCHEDULER
-            IdlePromise.RegisterIdle(commandData.Application);
-#endif
-
             InitializeAssemblies();
```

Comment:
[As you wished](https://github.com/DynamoDS/DynamoRevit/pull/132), @pboyer. Do have a look :)